sigmf-spec.md

```diff
@@ -106,6 +106,21 @@ file contains information that describes the dataset.
 3. The metadata file MUST have a `.meta` filename extension.
 4. The dataset file MUST have a `.data` filename extension.
 
+#### Archive Format
+
+The metadata and dataset files that comprise a SigMF recording may be combined
+into a file archive. Multiple SigMF recordings may be stored in a single
+archive.
```

Comment:
I think we should discuss the idea of combining multiple recordings separately before putting it in the spec. It is a UI/API complication for anything that wants to read a SigMF recording because you then need more than a file-picker, and people will be tempted to create little languages for representing pathnames-plus-item-in-the-archive, which gets messy.